Sources/Rendering/Core/Coordinate/api.md

```diff
@@ -20,15 +20,47 @@ The coordinate systems in vtk are as follows:
 
   POSE - world coords translated and rotated to the camera position and view direction
 
+  STABILIZED - used by rendering backends to deal with floating point resolution issues. Similar to world coordinates but with a translation to try to get the
```

Comment:
api.md are now generated (which mean we should start removing them from the repo).

The documentation should be added in the [TypeScript file](https://github.com/Kitware/vtk-js/blob/master/Sources/Rendering/Core/Coordinate/index.d.ts#L180-L210) instead.

Reply:
@daker is that true that the api.md are not auto-generated? If so, should we start removing them to prevent any confusion?

Reply:
Documentation needs to go in .d.ts file if it's present otherwise it needs to go in api.md, i will remove the api.md from folders where we have the .d.ts, i'll make a PR to cleanup this and sorry about the confusion.

Reply:
np, I just wanted to make sure. Can you grab the doc that Ken added here and put it inside the ts?

Reply:
Sure, i'll fix that.